.jenkins/dsl/jobs.groovy

```diff
@@ -195,7 +195,7 @@ void setupDeployJob(String jobFolder, KogitoJobType jobType) {
             }
 
             //Build branch name for quickstarts
-            stringParam('QUICKSTARTS_BUILD_BRANCH_NAME', 'development', 'Base branch for quickstarts. Set if you are not on a multibranch pipeline.')
+            stringParam('QUICKSTARTS_BUILD_BRANCH_NAME', "${GIT_BRANCH}" == 'master' ? 'development' : "${GIT_BRANCH}", 'Base branch for quickstarts. Set if you are not on a multibranch pipeline.')
```

Comment:
Let's not forget to add it to https://github.com/kiegroup/optaplanner/pull/1372.